doc/src/lexer_tutorial/index.md

```diff
@@ -3,6 +3,7 @@
 This part is about controlling the inner workings of LALRPOP's built-in lexer generator and using your own hand written parser.
 
 - [LALRPOP's lexer generator](001_lexer_gen.md)
-- [Writing a custom lexer](002_writing_custom_lexer.md)
-- [Using tokens with references](003_token_references.md)
+- [Lexing raw delimited content](002_raw_delimited_content.md)
+- [Writing a custom lexer](003_writing_custom_lexer.md)
+- [Using tokens with references](004_token_references.md)
 - [Using an external library](004_external_lib.md)
```

Comment:
Shouldn't this chapter be renumbered as well?

Reply:
Hunh, how did that get overlooked... will fix!

Reply:
Addressed by fe67e18b0e7ee2b685de3e67a38889ebf88fcfc9